packages/bootstrap/extra/dcos_internal_utils/cli.py

```diff
@@ -28,6 +35,41 @@ def wrapper(b, opts):
 def dcos_adminrouter(b, opts):
     b.cluster_id('/var/lib/dcos/cluster-id')
 
+    # Require the IAM to already be up and running. The IAM contains logic for
+    # achieving consensus about a key pair, and exposes the public key
+    # information via its JWKS endpoint. Talk directly to the local IAM instance
+    # which is reachable via the local network interface.
+    r = requests.get('http://127.0.0.1:8101/acs/api/v1/auth/jwks')
+
+    if r.status_code != 200:
+        log.info('JWKS retrieval failed. Got %s with body: %s', r, r.text)
+        sys.exit(1)
+
+    jwks = r.json()
+
+    # For now plan with the first key in the JWKS to correspond to the current
+    # private key used for signing authentiction tokens.
```

Comment:
I don't think this needs the "For now". Is this a valid assumption or not? If it is, then:
```
The first public key in the JWKS response corresponds to the current private key used for signing authentication tokens.
```

Reply:
done (in my local checkout)